rust/module.rs

```diff
@@ -155,12 +155,28 @@ fn build_modinfo_string_param(module: &str, field: &str, param: &str, content: &
         + &__build_modinfo_string_base(module, field, &content, &variable, false)
 }
 
+fn permissions_are_readonly(perms: &str) -> bool {
+    let (radix, digits) = if let Some(n) = perms.strip_prefix("0x") {
+        (16, n)
+    } else if let Some(n) = perms.strip_prefix("0o") {
+        (8, n)
+    } else if let Some(n) = perms.strip_prefix("0b") {
+        (2, n)
+    } else {
+        (10, perms)
+    };
+    match u32::from_str_radix(digits, radix) {
+        Ok(perms) => perms & 0o222 == 0,
+        Err(_) => false
+    }
+}
+
 /// Declares a kernel module.
 ///
 /// The `type` argument should be a type which implements the [`KernelModule`] trait.
 /// Also accepts various forms of kernel metadata.
 ///
-/// Example:
+/// ## Example
```

Comment:
I took a quick look and it seems the standard library uses the plural (and a top-level section):

```suggestion
/// # Examples
```